tests/conftest.py

```diff
@@ -34,12 +40,14 @@ def komodo_root(tmp_path_factory, python38_path):
 
     path = tmp_path_factory.mktemp("prog-res-komodo")
 
+    rhel_ver = rhel_version()
+
     # Install and configure pythons
     _install(python38_path, path / "2030.01.00-py38", ["numpy==1.18.4"])
     _install(python38_path, path / "2030.01.01-py38", ["numpy==1.19.1"])
     _install(python38_path, path / "2030.02.00-py38")
     _install(python38_path, path / "2030.03.00-py38-rhel9")
-    _install(python38_path, path / "bleeding-py38-rhel7")
+    _install(python38_path, path / f"bleeding-py38-rhel{rhel_ver}")
```

Comment:
Do we expect to have more versions here?
Or else we could omit the variable, although I'm unsure if this makes it less readable.

```
_install(python38_path, path / f"bleeding-py38-rhel{rhel_version()}")
```